src/permissions.rs

```diff
@@ -98,6 +98,7 @@ impl DenoPermissions {
     r
   }
 
+  /// Try to present the user with a permission prompt permission_denied if no_prompts is enabled
```

Comment:
Sorry - can you wrap this at 80 cols?

Odd that rustfmt isn’t doing that...

Also this will give us a chance to sample from the appveyor failure distraction again